.github/workflows/helm-ci-test.yml

```diff
@@ -0,0 +1,101 @@
+name: Test helm installation
+on:
+  push:
+    branches:
+      - main
```

Comment:
I don't think we need trigger on push to main (especially if we combine this with the existing `kind` test as I mention below.